lib/recursive_open_struct.rb

```diff
@@ -5,6 +5,7 @@ class RecursiveOpenStruct < OpenStruct
 
   def initialize(h=nil, args={})
     @recurse_over_arrays = args.fetch(:recurse_over_arrays,false)
+    @arrays = args.fetch(:arrays, false)
```

Comment:
I'd suggest making both inputs map to the same thing, making the input key `:arrays` set the same value as `:recurse_over_arrays`, rather than having them be two separate attributes that are treated the same way. Eg:

```
    @recurse_over_arrays = args.fetch(:recurse_over_arrays, args.fetch(:arrays, false))
```

Otherwise you are adding an unnecessary second instance variable and you then have to check both instance variables that both mean the same thing.